internal/models/mqtt.go

```diff
@@ -1,8 +1,10 @@
 package models
 
 type OutgoingMqttJson struct {
-	Bytes *string `json:"bytes,omitempty"`
-	Name  *string `json:"name,omitempty"`
-	Value any     `json:"value,omitempty"`
-	Unit  *string `json:"unit,omitempty"`
+	Bytes   *string `json:"bytes,omitempty"`
```

Comment:
While I don't necessarily mind adding fields to the structured JSON message, I would prefer to keep completely unrelated changes in separate PRs. It makes intention clearer and reviewing easier. This change isn't even mentioned in the description. 😄